src/index.js

```diff
@@ -57,6 +57,7 @@ const saveEdittedTask = (task) => {
 };
 
 const completedTask = (task) => {
+  window.alert('Task Completed');
```

Comment:
- [ ] [OPTIONAL] please remove the 2 `window.alert` it will make your application work better and your codebase will look a lot cleaner.

Reply:
Oh, I forgot removing that. Thanks for pointing it out :point_left: